account_mass_reconcile/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2012-2016 Camptocamp SA (Guewen Baconnier, Damien Crier, Matthieu Dietrich)
+# © 2010 Sébastien Beau
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Mass Reconcile",
+    "version": "9.0.1.3.1",
```

Comment:
As it is a portage the convention is to reset the version. Plus as it is a brand new and shiny module name it definitely can become 9.0.1.0.0